internal/pkg/config/config.go

```diff
@@ -151,6 +157,7 @@ func setConfigDefaults() {
 	viper.SetDefault(SessionTimeLimitKey, SessionTimeLimitDefault)
 	viper.SetDefault(IdentityProviderCustomEndpointKey, "")
 	viper.SetDefault(IdentityProviderCustomClientIdKey, "")
+	viper.SetDefault(AllowedUrlDomainKey, "")
```

Comment:
Couldn't we set the default value explicitly in the flag? Instead of living just in a global variable in the package?

Reply:
This would also make it more transparent to the user

Reply:
I changed the default value handling :)